roles/node_images_ncn_common/files/scripts/metal/metal-lib.sh

```diff
@@ -499,6 +504,8 @@ function install_csm_rpms() {
             | jq -r  '.items[] | .assets[] | .downloadUrl' | grep platform-utils | sort -V | tail -1)
         test -n "$iuf_cli_url" || iuf_cli_url=$(paginate "https://packages.local/service/rest/v1/components?repository=$repo" \
             | jq -r  '.items[] | .assets[] | .downloadUrl' | grep iuf-cli | sort -V | tail -1)
+        test -n "$cray_cmstools_url" || cray_cmstools_url=$(paginate "https://packages.local/service/rest/v1/components?repository=$repo" \
```

Comment:
Does this also provide `platform-utils`, or how does `platform-utils` get on the system now that it's removed from `suse.yml`?

Reply:
check out line 504.  we were already installing platform-utils this way.